src/workerd/io/tracer.c++

```diff
@@ -83,10 +83,11 @@ kj::Own<WorkerTracer> PipelineTracer::makeWorkerTracer(PipelineLogLevel pipeline
     kj::Maybe<kj::String> dispatchNamespace,
     kj::Array<kj::String> scriptTags,
     kj::Maybe<kj::String> entrypoint,
-    kj::Maybe<kj::Own<tracing::TailStreamWriter>> maybeTailStreamWriter) {
```

Comment:
nit: maybeTailStreamWriter should continue to be the last argument (i.e. all metadata parameters including durableObjectId should come before it)